lib/src/util/condition_scope_visitor.dart

```diff
@@ -6,8 +6,8 @@ import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/token.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:analyzer/dart/element/element.dart';
-import 'package:analyzer/src/generated/resolver.dart';
 import 'package:linter/src/util/dart_type_utilities.dart';
+import 'package:linter/src/analyzer.dart';
```

Comment:
Sort this before `package:linter/src/util/..`?

Reply:
Done